### PR TITLE
Mantener credenciales de usuario

### PR DESCRIPTION
**Pull request by AlbaFrancisco**
depuracion de errores de mantener credenciales

### CODE REVIEW

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/managers/SharedPrefsManager.java [hunk 1]
```diff
@@ -28,18 +28,29 @@ public static SharedPrefsManager getInstance(Context context) {
...
+    }
+
     public void remove(String key) {
         final SharedPreferences.Editor editor = this.sharedPreferences.edit();
         editor.remove(key);
         editor.commit();
     }
 
 
+
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/managers/SharedPrefsManager.java]: unneded empty space

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 2]
```diff
@@ -41,6 +47,7 @@ protected void onCreate(Bundle savedInstanceState) {
         mUsername = (EditText) findViewById(R.id.userLoginText);
         mPassword = (EditText) findViewById(R.id.passLoginText);
         mLoginButton = (Button) findViewById(R.id.login_button);
+        mRemember=(CheckBox) findViewById(R.id.remember);
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: different code style

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 3]
```diff
@@ -67,7 +74,12 @@ private void doLogin() {
             @Override
             public void onResponse(LoginModel response) {
                 String token = response.getToken();
+                if(check==true){
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: different code style

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 4]
```diff
@@ -97,7 +109,9 @@ public void onClickClose() {
                                                         onLoginSuccess,
                                                         onLoginError);
 
+
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: too much empty space

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 5]
```diff
@@ -112,11 +126,20 @@ private void hideLoginInProgress() {
...
     }
 
     private void showAuthenticationError() {
         Toast.makeText(LoginActivity.this, "Incorrect login", Toast.LENGTH_LONG).show();
     }
+
+    public void  clickRemember(View v) {
+        String s = "Estado: " + (mRemember.isChecked() ? "Marcado" : "No Marcado");
+        Toast.makeText(this, s, Toast.LENGTH_LONG).show();
+        check=mRemember.isChecked();
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: different code style

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 6]
```diff
@@ -112,11 +126,20 @@ private void hideLoginInProgress() {
...
         intent.putExtra(MainActivity.EXTRA_TOKEN, token);
         startActivity(intent);
     }
 
     private void showAuthenticationError() {
         Toast.makeText(LoginActivity.this, "Incorrect login", Toast.LENGTH_LONG).show();
     }
+
+    public void  clickRemember(View v) {
+        String s = "Estado: " + (mRemember.isChecked() ? "Marcado" : "No Marcado");
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: untranslated string

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 7]
```diff
@@ -3,18 +3,30 @@
...
 
     public static final String EXTRA_TOKEN = "token";
-
+    String token;
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        SharedPrefsManager remember = SharedPrefsManager.getInstance(getApplicationContext());
+        String prefs=remember.getString("remember");
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: different code style

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 8]
```diff
@@ -3,18 +3,30 @@
...
+        String TAG = "MyActivity";
+        Log.e(TAG, prefs);
+        if(!prefs.equals(null)){
+             token=prefs;
+             showSurveyListPage(token);
+            finish();
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: different indentation

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 9]
```diff
@@ -3,18 +3,30 @@
...
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        SharedPrefsManager remember = SharedPrefsManager.getInstance(getApplicationContext());
+        String prefs=remember.getString("remember");
+        String TAG = "MyActivity";
+        Log.e(TAG, prefs);
+        if(!prefs.equals(null)){
+             token=prefs;
```
Comment (jvillafanez) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: different code style

File: Surveyx/app/src/main/res/layout/activity_login.xml [hunk 10]
```diff
@@ -138,4 +159,6 @@
             android:layout_centerHorizontal="true" />
 
     </RelativeLayout>
+
```
Comment (jvillafanez) [inline Surveyx/app/src/main/res/layout/activity_login.xml]: too much empty space

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/managers/SharedPrefsManager.java (hunk 1 above)
Comment (lrevilla) [reply Surveyx/app/src/main/java/es/academy/solidgear/surveyx/managers/SharedPrefsManager.java]: +1

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java (hunk 2 above)
Comment (lrevilla) [reply Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: aka whitespaces between before and after "="

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java (hunk 3 above)
Comment (lrevilla) [reply Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: this check can be performed also as `if (check) {`

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java (hunk 4 above)
Comment (lrevilla) [reply Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: +1

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 11]
```diff
@@ -119,4 +133,9 @@ private void openMainActivity(String token) {
     private void showAuthenticationError() {
         Toast.makeText(LoginActivity.this, "Incorrect login", Toast.LENGTH_LONG).show();
     }
+
+    public void  clickRemember(View v) {
+        check=mRemember.isChecked();
```
Comment (lrevilla) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: whitespaces between before and after "="

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 12]
```diff
@@ -3,18 +3,30 @@
 import android.app.Activity;
 import android.content.Intent;
 import android.os.Bundle;
+import android.util.Log;
```
Comment (lrevilla) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: not needed import

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java [hunk 13]
```diff
@@ -1,10 +1,13 @@
 package es.academy.solidgear.surveyx.ui.activities;
 
 import android.content.Intent;
+import android.content.SharedPreferences;
 import android.graphics.Typeface;
 import android.os.Bundle;
+import android.util.Log;
```
Comment (lrevilla) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/LoginActivity.java]: not needed import

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 14]
```diff
@@ -3,18 +3,30 @@
...
 
 import es.academy.solidgear.surveyx.R;
+import es.academy.solidgear.surveyx.managers.SharedPrefsManager;
 
 
 public class MainActivity extends Activity {
 
     public static final String EXTRA_TOKEN = "token";
-
+    String token;
```
Comment (lrevilla) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: You can instantiate this variable inside the `onCreate` method as it is not needed anywhere else

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java [hunk 15]
```diff
@@ -3,18 +3,30 @@
...
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        SharedPrefsManager remember = SharedPrefsManager.getInstance(getApplicationContext());
+        String prefs=remember.getString("remember");
+        String TAG = "MyActivity";
+        Log.e(TAG, prefs);
```
Comment (lrevilla) [inline Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: Log

File: Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java (hunk 8 above)
Comment (lrevilla) [reply Surveyx/app/src/main/java/es/academy/solidgear/surveyx/ui/activities/MainActivity.java]: +1

**Pull request merged**